Documentation/contributor-guide/triage_issues.md

```diff
@@ -48,9 +48,42 @@ Adding a `type` label to an issue helps create visibility on the health of the p
 
 ### Support requests
 
-Some people mistakenly use GitHub issues to file support requests. Usually they are asking for help operating or configuring some aspect of etcd. In general we prefer that support requests are raised as [discussions](https://github.com/etcd-io/etcd/discussions), however if you identify that an issue is a support request please add the `type/support` or `type/question` label.
+As a general rule the focus for etcd support is to address common themes in a broad way that helps all users, i.e. through channels like known issues, frequently asked questions and high quality documentation. To make the best use of project members time we should avoid providing 1:1 support if a broad approach is available.
 
-When responding to these requests please keep in mind the etcd [supported versions policy](https://etcd.io/docs/v3.6/op-guide/versioning/#service-versioning-and-supported-versions). If the request relates to an unsupported version and will require significant effort please instead kindly make the issue creator aware of the etcd support policy, encourage they update to a supported release, then close the issue, refer to [this example](https://github.com/etcd-io/etcd/issues/15796).
+Some people mistakenly use our GitHub bug report or feature request templates to file support requests. Usually they are asking for help operating or configuring some aspect of etcd. Support requests for etcd should instead be raised as [discussions](https://github.com/etcd-io/etcd/discussions).
+
+Common types of support requests are:
+
+ 1. Questions about configuring or operating existing well documented etcd features, for example <https://github.com/etcd-io/etcd/issues/15836>. Note - If an existing feature is not well documented please apply the `area/documentation` label and propose documentation improvements that would prevent future users from stumbling on the problem again.
+
+ 2. Bug reports or questions about unspported versions of etcd, for example <https://github.com/etcd-io/etcd/issues/15796>. When responding to these issues please refer to our [supported versions documentation](https://etcd.io/docs/latest/op-guide/versioning) and encourage the reporter to upgrade to a supported version as soon as possible. We should limit the effort supporting users that do not make the effort to run a supported version of etcd.
```

Comment:
I'm missing the differentiation of minor vs patch version. We should not support minor versions outside of supported window, but also ask users to upgrade/reproduce the issue on the latest patch versions within a supported window. As a contributor I don't want to read changelog on behalf of user that reported a bug that might have been already fixed on newer patch version.

Reply:
With such long lived minor release versions I agree a mention to patch version would be useful. As a starting point I have updated to:

> When responding to these issues please refer to our [supported versions documentation](https://etcd.io/docs/latest/op-guide/versioning) and encourage the reporter to upgrade to a recent patch release of a supported version as soon as possible. We should limit the effort supporting users that do not make the effort to run a supported version of etcd or ensure their version is patched.

I've left the definition as "recent" for patch release as expecting every bug report to be raised against latest patch release is maybe a bit strong?  

If you think we should request an exact version, i.e. latest, or n-x please let me know.